docs/architecture/hotspot-makers/become-a-maker/basic-miner-operation.mdx

```diff
@@ -84,15 +88,12 @@ Log in and out of your account to apply these changes. You are now ready to use
 
 ### Port Forwarding
```

Comment:
We should remove the entire port forwarding section. 1680 is used for communication between the packer forwarder and gateway-rs, which happens locally and doesn't need to be exposed to the open internet

Reply:
All removed